chromadb/test/property/test_embeddings.py

```diff
@@ -799,10 +799,10 @@ def test_autocasting_validate_embeddings_for_compatible_types(
     validated_embeddings = validate_embeddings(Collection._normalize_embeddings(embds))
     assert all(
         [
-            isinstance(value, list)
+            isinstance(value, np.ndarray)
             and all(
                 [
-                    isinstance(vec, (int, float)) and not isinstance(vec, bool)
+                    isinstance(vec, (np.integer, np.floating)) and not isinstance(vec, bool)
```

Comment:
I think it's sufficient to check the dtype of the np array here instead of checking every single element (same below)